### DISCUSSION TITLE
500:Internal server error

### DISCUSSION THREAD

**Discussion by venom-10** [vercel/community#2416] [Help] [answered]
### Summary

![Screenshot (240)](https://github.com/vercel/community/assets/96457403/b5fd02cf-5df1-454d-99c1-b86ff5c07c60)

I want to fetch the data from MongoDB Server using api but the size of data is like 10mb andthis error happened everytime. 

### Example

https://org-connect-2xn5.vercel.app/
this error -> https://org-connect-2xn5.vercel.app/getData

**Comment by swarnava** [comment, marked as answer]
The url says Deployment not found. Have you removed your project? If you are on Hobby plan, you need to make sure the API return response in 10 Seconds: https://vercel.com/docs/concepts/limits/overview#edge-middleware-and-edge-functions-execution-timeout
